index.html

```diff
@@ -802,6 +802,11 @@ <h3>Link Relation Types</h3>
               <td>link target is a collection member of the thing or thing model.</td>
               <td></td>
             </tr>
+            <tr>
+              <td><code>alternate</code></td>
+              <td>link target is a HTML based user interface of the thing.</td>
```

Comment:
```suggestion
              <td>link target is an alternative representation of the Thing.</td>
```

link rel=alternative could also be used for other use cases, such as linking to an XML version of the Thing Description for example. It's only when the type is text/html that it SHOULD be interpreted as a user interface.

Reply:
It can also be a documentation website.

Reply:
@egekorkan That's what the `service-doc` link relation is used for. There's an assertion which specifically says that link rel=alternative with type=text/html should be interpreted as a user interface, but a different content type could be used to mean something else.